keys/employees-keys/README.md

```diff
@@ -116,10 +133,159 @@ sub   4096R/CC8BE1D5 2021-04-02
 
 The key ID in this case is `7429663E`.
 
-## How to export your key
+### Sending key for signing
+
+Generate ASCII armor public key file:
 
 ```
 gpg --armor --output your-name-key.asc --export "YOUR KEY ID HERE"
 ```
 
-Share `your-name-key.asc` textual file as the public portion of your key.
+Obtain fingerprint of your key:
+
+```
+gpg -n -q --import --import-options import-show your-name-key.asc
```

Comment:
Won't something simpler like `gpg --fingerprint your.name@3mdeb.com` do?

Reply:
@SergiiDmytruk doesn't work for me with gpg v2.2.12